tests/integration/state.go

```diff
@@ -132,10 +132,17 @@ type state struct {
 	// Identities by node index, by identity index.
 	identities [][]identity.Identity
 
-	// Will recieve an item once all actions have finished processing.
+	// identitiesByName
+	identitiesByName map[string]identity.Identity
+
+	// The seed for the next node identity generation. It starts at 0x7fffffff to avoid
+	// collisions with the default identities.
+	nextNodeIdentityGenSeed int
```

Comment:
todo: can you describe the purpose? I don't quite understand why this is needed?

Reply:
I added some more docs. Not sure what else to add.